src/ui_dialogs.ts

```diff
@@ -119,6 +145,78 @@ export function openEditLinkDialog(column: LinkColumn, $header: d3.Selection<Col
   });
 }
 
+
+function removePop(popup, rendererContent, currentElement) {
```

Comment:
+ the name `removePop` can be improved to better express the functionality, since it doesn't actually immediatelly remove the popup